tests/test_nanoevents_vector.py

```diff
@@ -539,17 +539,17 @@ def test_inherited_method_transpose(lcoord, threecoord, twocoord):
             behavior=vector.behavior,
         )
 
-    assert_record_arrays_equal(a + b, b + a, check_type=True)
-    assert_record_arrays_equal(a + c, c + a, check_type=True)
-    assert_record_arrays_equal(b + c, c + b, check_type=True)
+    assert_record_arrays_equal(a.like(b) + b, b + a.like(b), check_type=True)
+    assert_record_arrays_equal(a.like(c) + c, c + a.like(c), check_type=True)
+    assert_record_arrays_equal(b.like(c) + c, c + b.like(c), check_type=True)
```

Comment:
We finally see the type errors on adding 2 vectors of different geometric dimensions as now this is handled on the scikit-hep/vector side.